crates/claims/crates/vc/src/syntax/credential.rs

```diff
@@ -14,7 +14,7 @@ use crate::{v1, v2, MaybeIdentified};
 ///
 /// If you care about required context and/or type, use the
 /// [`AnySpecializedJsonCredential`] type directly.
-pub type AnyJsonCredential<S = json_syntax::Value> = AnySpecializedJsonCredential<S>;
+pub type AnyJsonCredential<S = json_syntax::Object> = AnySpecializedJsonCredential<S>;
```

Comment:
I would be tempted to say this should be `NonEmptyObject` but I don't feel strongly about it

Reply:
I didn't see the same non-emptiness constraint for VCDM v1

Reply:
Ah you mean for the union of v1 and v2. I don't have a strong opinion either. I decided to go for completeness over soundness.